Source/Core/Core/PowerPC/JitArm64/JitArm64_RegCache.cpp

```diff
@@ -222,7 +222,7 @@ void Arm64GPRCache::GetAllocationOrder()
 	const std::vector<ARM64Reg> allocation_order =
 	{
 		// Callee saved
-		W28, W27, W26, W25, W24, W23, W22, W21, W20,
```

Comment:
Yes, since it's a callee saved register if a function wants to use that register it first has to save it on to the stack and then put it back when done.